### PR TITLE
Lesson 1.2 submission

### PR DESCRIPTION
**Pull request by duokobia**
This is the week2 submission.

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const AddTodoForm = () => {
+  return (
+    <form>
+        <label>Title</label>
+        <input htmlFor="todoTitle" id="todoTitle"></input>
```
Comment (kramduckner) [inline src/AddTodoForm.js]: The htmlFor attribute needs to be placed in the label element! That's how the html links the label and input. It'll still work like this, but accessibility tools for users with visual impairments will have a harder time understanding the form.

**Review by kramduckner — CHANGES_REQUESTED** [review]
Looking great @duokobia ! Just one small request.

File: src/AddTodoForm.js (hunk 1 above)
Comment (duokobia) [reply src/AddTodoForm.js]: Hi Kram,

Thank you for the review.

I have made the adjustment as requested please.

Kind regards.

**Review by kramduckner — APPROVED** [review]
Looks great!

**Comment by duokobia** [comment]
Thank you very much Mark.

I appreciate.

**Pull request merged**